### ISSUE TITLE
Listing commands per guilds

### ISSUE THREAD

**Issue by ririko5834** [Androz2091/slash-commands-gui#7]
Hi, can you add that when there are some guild based commands registred, there will be guild picker for that guilds in settings?

**Comment by Androz2091** [comment]
Hello, you can already enter a guild ID right?

**Comment by ririko5834** [comment]
Yes, but when you don't remember for what guilds you registred slash commands and want to delete it. There should be something to get list of all registred slash commands and manage it

**Comment by Androz2091** [comment]
I don't think there is an endpoint to get all the commands in all the guilds. Or we would need to loop over all the guilds to get their commands, but I don't think it is a good idea 🤔

**Comment by danshilm** [comment]
Is it possible to have a dropdown with the guilds that the bot is in?

**Comment by Androz2091** [comment]
We would need the bot's token instead 🤔 
Maybe yes even if it's not as secure as an oauth token, it's better to provide token authentication

**Comment by ririko5834** [comment]
You should make that you can use client secret or bot token. Because someone also me have oauth application for slash commands without bot

**Comment by D3ord3NidAm** [comment]
As the bot joins a server, save the guild name and guild id to a json file or database so you don't have to worry about it. Use a function to get the specific guild you want to use it for and pull into the command.
